### PR TITLE
[JENKINS-38987 Follow-up] Expose the SCM API pronoun information

### PR DESCRIPTION
**Pull request by stephenc**
@reviewbybees

### CODE REVIEW

**Comment by abayer** [comment]
:bee:

File: pom.xml [hunk 1]
```diff
@@ -69,7 +69,7 @@ THE SOFTWARE.
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-job</artifactId>
-            <version>2.6</version>
+            <version>2.8-SNAPSHOT</version>
```
Comment (jglick) [inline pom.xml]: Now unnecessary AFAICT

File: src/main/java/org/jenkinsci/plugins/workflow/multibranch/BranchJobProperty.java [hunk 2]
```diff
@@ -82,4 +85,20 @@ synchronized void setBranch(@Nonnull Branch branch) {
 
     }
 
+    @Extension
+    public static class AlternativeUiTextProviderImpl extends AlternativeUiTextProvider {
+
+        @Override
+        public <T> String getText(Message<T> text, T context) {
+            if (text == AbstractItem.PRONOUN && context instanceof WorkflowJob) {
+                WorkflowJob job = WorkflowJob.class.cast(context);
```
Comment (jglick) [inline src/main/java/org/jenkinsci/plugins/workflow/multibranch/BranchJobProperty.java]:
```java
WorkflowJob job = (WorkflowJob) context;
```

File: pom.xml [hunk 3]
```diff
@@ -79,22 +79,22 @@ THE SOFTWARE.
...
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>scm-api</artifactId>
-            <version>1.3</version>
+            <version>1.4-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>branch-api</artifactId>
-            <version>1.11</version>
+            <version>1.12-SNAPSHOT</version>
```
Comment (jglick) [inline pom.xml]: ditto

File: pom.xml [hunk 4]
```diff
@@ -79,22 +79,22 @@ THE SOFTWARE.
...
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>cloudbees-folder</artifactId>
-            <version>5.12</version>
+            <version>5.14-SNAPSHOT</version>
```
Comment (jglick) [inline pom.xml]: ditto

File: pom.xml [hunk 5]
```diff
@@ -79,22 +79,22 @@ THE SOFTWARE.
...
         </dependency>
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-api</artifactId>
-            <version>2.3</version>
+            <version>2.5-SNAPSHOT</version>
```
Comment (jglick) [inline pom.xml]: ditto

File: pom.xml [hunk 6]
```diff
@@ -140,7 +140,7 @@ THE SOFTWARE.
         <dependency>
             <groupId>org.jenkins-ci.plugins.workflow</groupId>
             <artifactId>workflow-support</artifactId>
-            <version>2.4</version>
+            <version>2.9-SNAPSHOT</version>
```
Comment (jglick) [inline pom.xml]: ditto

File: pom.xml [hunk 7]
```diff
@@ -218,7 +218,7 @@ THE SOFTWARE.
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>credentials</artifactId>
-            <version>2.1.4</version>
+            <version>2.1.6</version>
```
Comment (jglick) [inline pom.xml]: ?

File: pom.xml (hunk 3 above)
Comment (stephenc) [inline pom.xml]: This is the version that depends on scm-api 1.4-SNAPSHOT, I want to keep dependencies aligned

File: pom.xml (hunk 7 above)
Comment (stephenc) [inline pom.xml]: Keeping dependency version aligned with highest version in transitive dependencies

**Comment by rsandell** [comment]
:bee:

**Comment by stephenc** [comment]
@jglick ping

**Pull request merged**